src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -124,6 +131,7 @@ public void onLoop(double timestamp)
                     case SHOOTING_BAY:
                         if (mStateChanged)
                         {
+                            mShootTimer.reset();
                             mShootTimer.start();
                             mRampSolenoid.set(Constants.kRampSolenoidRetract);
                             mRampMotor.set(ControlMode.PercentOutput, Constants.kShootSpeed);
```

Comment:
This is actually below - but when we move into the HOLDING state, we should call `mShootTimer.stop();` and `mShootTimer.reset()`

Reply:
That actually happens on every state change so this shouldn't be necessary.